resources/schema.py

```diff
@@ -53,6 +55,25 @@ def _resolve_piers(**kwargs):
         ["min_berth_width" in kwargs, "min_berth_length" in kwargs]
     )
 
+    if filter_empty_berths and application_global_id:
```

Comment:
could we rename `filter_empty_berths ` ->  ~`filter_dimensions_given`? it gets a bit confusing now with more logic here 😅

Reply:
Trying to come up with a clearer name,  since using `filter_dimensions_given` might make [L92](https://github.com/City-of-Helsinki/berth-reservations/pull/146/files#diff-09f1721fc193a5a442fee1cf24f5bd49R92) a bit confusing.

Reply:
`filter_by_dimensions`?

Reply:
`has_dimensions_filter`?

Reply:
yeah, that works 👍

Reply:
:happy-cat: